content/kubernetes/release-notes/6-2-18-releases/k8s-6-2-18-3.md

```diff
@@ -21,6 +21,11 @@ The key bug fixes, new features, and known limitations are described below.
 If you use NGINX as an ingress controller for Redis Enterprise, **do not upgrade** to the 6.2.18-3 release. Skip this release and upgrade to version [6.2.18-41]({{<relref "/kubernetes/release-notes/6-2-18-releases/">}}) instead.
 {{</warning>}}
 
+{{<warning>}} Due to a change in the SCC, on OpenShift clusters running version 6.2.12 or earlier upgrading to version 6.2.18 or later, where `node:1` is <b>not</b> the master node, the upgrade might get stuck.
+
+For more info and steps to prevent this issue, see [upgrade a Redis Enterprise cluster (REC)]({{<relref "/kubernetes/re-clusters/upgrade-redis-cluster.md#before-upgrading">}}).
```

Comment:
```suggestion
For more info and steps to prevent this issue, see [upgrade a Redis Enterprise cluster (REC)]({{<relref "/kubernetes/re-clusters/upgrade-redis-cluster#before-upgrading">}}).
```